tests/contrib/django/django1_app/urls.py

```diff
@@ -1,10 +1,15 @@
-from django.conf.urls import url
+import django
 from django.views.decorators.cache import cache_page
 from django.views.generic import TemplateView
 
 from .. import views
 
 
+if django.VERSION < (2, 0, 0):
+    from django.conf.urls import url
+else:
+    from django.urls import re_path as url
```

Comment:
we probably shouldn't do this, don't want to confuse anyone looking at all of the `urlpatterns` and expecting `django.conf.urls.url` instead of `re_path`.

Reply:
this is also a `django` 1.x app, why do we need this change?

Reply:
I think I'd prefer to just import `re_path` and use that instead rather than mapping it to `url`